samples/openthread/coap_client/overlay-multiprotocol_ble.conf

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Enable multiprotocol service layer
```

Comment:
Strictly speaking there is no need to select MPSL when BT_LL_SOFTDEVICE is selected. If you intend to do so for clarity, I'm fine with that

Reply:
Yes, by explicitly enabling MPSL I wanted to emphasize that MPSL is a core module and without it, the multiprotocol functionality, will not work.